distribution/kubernetes/vector-agent/rbac.yaml

```diff
@@ -6,6 +6,11 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: vector
+  labels:
+    app.kubernetes.io/name: vector
+    app.kubernetes.io/instance: vector
+    app.kubernetes.io/component: Agent
+    app.kubernetes.io/version: "0.19.0-distroless-libc"
```

Comment:
Without knowing the context, just checking if `0.19.0` is intended here?

Reply:
It ought to be 0.20.0... or maybe the install instructions have the user override it regardless to avoid a chicken & egg situation between these and the helm charts.

Reply:
Yeah, it is since these files are generated from the `helm-charts` which are still using 0.19.0. It's a bit of a chicken-and-egg situation with new vector releases. After the 0.20 release, I'll update the helm-charts and regenerate these assets.

Reply:
Technically it's fine given the usage instructions we have - which do override the image so users won't get the wrong version... but :awkward:

Reply:
Hmm, yeah, that's true, we could have the generation script update to the new version before the images are published.